docs/projects/ad7616_sdz/index.rst

```diff
@@ -249,8 +250,8 @@ axi_ad7616 *    10  54         87
 .. admonition:: Legend
    :class: note
 
-   - ``*`` instantiated only for SER_PAR_N=0 (parallel interface)
-   - ``**`` instantiated only for SER_PAR_N=1 (serial interface)
+   - ``*`` instantiated only for INTF=0 (parallel interface)
+   - ``**`` instantiated only for INTF=1 (serial interface)
 
 Building the HDL project
```

Comment:
Specify somewhere in this section as well, which is the default configuration

Reply:
Done.